examples/amp-fx-collection.html

```diff
@@ -0,0 +1,319 @@
+<!--
```

Comment:
Might submit this as an example actually. Without the large DO NOT SUBMIT ASCII art that is

Reply:
sg!